### PR TITLE
adjust api according feedback

### PR DESCRIPTION
**Pull request by super626**
Animation3D::getOrCreate to Animation3D::create
Animate3D setSpeed support negative value, which means play reverse
Animate3D  remove get or set PlayBack which is confusing.
Animate3D add parameter check for setWeight

Deprecated old ones

### CODE REVIEW

**Comment by minggo** [comment]
Please deprecated old one and add new one.

**Comment by minggo** [comment]
[ci rebuild]

**Comment by super626** [comment]
OK

File: cocos/3d/CCAnimate3D.h [hunk 1]
```diff
@@ -66,17 +67,17 @@ class Animate3D: public ActionInterval
     
     virtual void update(float t) override;
     
-    /**get & set speed */
-    float getSpeed() const { return _speed; }
-    void setSpeed(float speed) { _speed = speed; }
+    /**get & set speed, negative speed means playing reverse */
+    float getSpeed() const;
+    void setSpeed(float speed);
```
Comment (minggo) [inline cocos/3d/CCAnimate3D.h:72]: Please add doxygen style comment for this function. Especially  mentioned this function https://github.com/cocos2d/cocos2d-x/blob/v3/cocos/2d/CCNode.h#L287

File: cocos/3d/CCAnimate3D.h [hunk 2]
```diff
@@ -66,17 +67,17 @@ class Animate3D: public ActionInterval
...
     float getWeight() const { return _weight; }
-    void setWeight(float weight) { _weight = weight; }
+    void setWeight(float weight);
     
-    /**get & set play back*/
-    bool getPlayBack() const { return _playBack; }
-    void setPlayBack(bool playBack) { _playBack = playBack; }
+    /**get & set play reverse*/
+    CC_DEPRECATED_ATTRIBUTE bool getPlayBack() const { return _playReverse; }
+    CC_DEPRECATED_ATTRIBUTE void setPlayBack(bool reverse) { _playReverse = reverse; }
```
Comment (minggo) [inline cocos/3d/CCAnimate3D.h:80]: Please add comment to describe use which function instead.

**Comment by minggo** [comment]
[ci rebuild]

**Pull request merged**